invenio_communities/communities/services/config.py

```diff
@@ -32,18 +33,7 @@ class SearchOptions(SearchOptionsBase):
     """Search options."""
 
     facets = {
-        'type': TermsFacet(
-            field='metadata.type',
-            label=_('Type'),
-            value_labels={
-                'organization': _('Organization'),
-                'event': _('Event'),
-                'topic': _('Topic'),
-                'project': _('Project'),
-            }
-        ),
-        'domain': TermsFacet(field='metadata.domains'),
```

Comment:
In the mockups there are 2 facets and you only keep one, was this removed by mistake or on purpose?

Reply:
The domain was already defined as a facet but the field is not implemented at all. In the mockups, we have facets for 
- Visibility
- Ownership

I implemented the first one but the second one I think is not there yet. I updated the screenshot of how it looks like now